src/client/session/test.rs

```diff
@@ -389,7 +389,7 @@ async fn session_usage() {
         F: Fn(EventClient) -> G,
         G: Future<Output = ()>,
     {
-        let client = EventClient::new().await;
+        let client = Client::test_builder().monitor_events().build().await;
```

Comment:
This is a little more verbose, but only slightly, and I think that's okay because it makes it easy to include other builder setup and removes the temptation for further constructors in `EventClient`.